### DISCUSSION TITLE
Only who participant epoch3 can have an option for frozen seeds in epoch4 registration form

### DISCUSSION THREAD

**Discussion by kevingzhang** [tearust/teaproject#73] [Ideas]
Add one question in the epoch4 registration form: "Are you one of the participants of epoch3?". If the answer is yes, you can get the selection of frozen seeds. If the answer is no, you will not allow to choose those frozen seeds option.

**Comment by geopor** [comment]
Currently, all people who have already signed up for epoch 4 have participated in a previous contest. When epoch 4 starts it will be like epoch 3, they have to message on Telegram to get an invite which doesn't include frozen seeds. So I don't think the registration form should change - most people signing up while the form is still open will have already participated, and those few extra who didn't can get some benefit for signing up early having found out about the contest through Telegram or another social channel.
